File: WebApplication/src/modules/student/dtos/person-lot.dtos/person-student.dtos/person-student-diploma.dto.ts
import { PersonStudentDiplomaFile } from "./person-student-diploma-file.dto";

export class PersonStudentDiplomaDto {
  diplomaNumber: string;
  registrationDiplomaNumber: string;
  diplomaDate: Date | null;
  file: PersonStudentDiplomaFile;
  isValid: boolean;
}

File: StudentCard.Application/Students/Dtos/PersonLotDtos/PersonStudentDtos/PersonStudentDiplomaDto.cs
using StudentCard.Data.Rdpzsd.Students.Parts;
using System;

namespace StudentCard.Application.Students.Dtos.PersonLotDtos.PersonStudentDtos
{
    public class PersonStudentDiplomaDto
    {
        public string DiplomaNumber { get; set; }
        public string RegistrationDiplomaNumber { get; set; }
        public DateTime? DiplomaDate { get; set; }

        public PersonStudentDiplomaFile File { get; set; }

        public bool IsValid { get; set; } = true;
    }
}
